### ISSUE TITLE
only public and localization folders

### ISSUE THREAD

**Issue by Gentleman358** [Byte0Bard/BG3-Equipment-Generator#5]
when I press "Generate Equipment" it only makes "Public" and "Localization" folders, Public folder have some subfolders with lsf files, but there is no models folder, only one time from 5 attempts it made a real mod

**Comment by Byte0Bard** [comment]
Are you still having this issue?

**Comment by Gentleman358** [comment]
well, I tried it like a week ago for last time, but I think yes, when I tried to make a trident it was making only 2 folders

**Comment by Byte0Bard** [comment]
Would you be able to send your log files and your project folder that was generated? You can find where the log files are by clicking the gear button in the generator. This will open a settings panel. Then, click the button that says "Open Save Folder." It will open the folder that contains the "logs" folder.

**Comment by Gentleman358** [comment]
[adw.zip](https://github.com/Byte0Bard/BG3-Equipment-Generator/files/15191190/adw.zip)
[console-2024-05-03-02-26-09.log](https://github.com/Byte0Bard/BG3-Equipment-Generator/files/15191193/console-2024-05-03-02-26-09.log)

**Comment by Gentleman358** [comment]
these I made just now, so it may not match my first attempts, but it still didn't make any textures or model in generated folder

**Comment by Byte0Bard** [comment]
Yeah it didn't make anything. Would you be able to send me your .json save file?

**Comment by Gentleman358** [comment]
I.. don't have one, I think I didn't created it
